ext/nginx.conf.example

```diff
@@ -1,10 +1,6 @@
 # NOTE: if you wish to only connect using fallback, you can
 # remove all ssl related content from the configuration.
 
-upstream bancho {
```

Comment:
why is this removed when its still used further down?

Reply:
i think http;//bancho would be valid within the docker compose so this is no longer needed